### PR TITLE
Dayodeji/auth context branch

### PR DESCRIPTION
**Pull request by DidunAyodeji**
Made some changes to improve dev experience

### CODE REVIEW

File: 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md [hunk 1]
```diff
@@ -123,19 +123,25 @@ Open the project in your IDE to configure the code.
 
 ### 3. Setup the MFA Conditional Access Context
 
-1. In the Azure Portal, go to the Microsoft Entra Conditional Access section.
+1. In the Azure Portal, type Microsoft Entra Conditional Access in the searchbar and select it.
```
Comment (jmprieur) [inline 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md:126]:
```suggestion
1. In the Azure Portal, type Microsoft Entra Conditional Access in the search bar and select it.
```

File: 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md [hunk 2]
```diff
@@ -123,19 +123,25 @@ Open the project in your IDE to configure the code.
...
 
 ![CA](./ReadmeFiles/CA.png)
 
 Note: this is a different Azure Portal section than Microsoft Entra ID
 
-2. Go to the "Authentication Contexts" page, and create a new context. Give it a name (e.g. "MFA") and select and id from the list, for example "c1". Ensure the "Publish To Apps" checkbox is checked. Make a note of the ID.
+2. Go to the `Authentication Contexts` page, and click on `New Authentication context`. Give it a name (e.g. "MFA") and select an ID from the list, for example `c1`. Ensure the "Publish To Apps" checkbox is checked. Make a note of the ID.
```
Comment (jmprieur) [inline 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md:132]: ID of the auth context?

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @DidunAyodeji

**Review by bgavrilMS — APPROVED** [review]

File: 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md (hunk 2 above)
Comment (DidunAyodeji) [reply 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md:132]: correct!

Comment (DidunAyodeji) [reply 3-java-servlet-web-app/1-Authentication/sign-in-auth-context/README.md:132]: I'll add that to be clearer thank you!

**Pull request merged**